subsys/partition_manager/pm.yml.modem_trace

```diff
@@ -0,0 +1,6 @@
+#include <autoconf.h>
+
+modem_trace:
+  placement: {before: [end]}
```

Comment:
Have you tried dropping this? I would expect that its not needed for the external flash region (not sure though, so just test it out if you have time)

Reply:
It's a bit short time, so I think this has to stay for now. It should also work in other scenarios with multiple partitions on the external flash, so wouldn't pm need the `placement` to order the partitions?